lib/workers/repository/updates/generate.ts

```diff
@@ -73,13 +73,17 @@ export function generateBranchConfig(
   const depNames: string[] = [];
   const newValue: string[] = [];
   const toVersions: string[] = [];
+  const toValues: string[] = [];
```

Comment:
I think we should use a `Set<string>` here instead.

Reply:
adapted, should I still check via `has` or just call `add` without checking? sorry, still not that familiar with some JS things :)